src/sidebar/store/modules/test/selection-test.js

```diff
@@ -64,9 +64,11 @@ describe('sidebar/store/modules/selection', () => {
       assert.isTrue(store.hasAppliedFilter());
     });
 
-    it('returns true if user-focused mode applied', () => {
-      store = createStore([selection], [{ focus: { user: {} } }]);
-      store.toggleFocusMode(true);
+    it('returns true if user-focused mode is configured', () => {
+      store = createStore(
+        [selection],
+        [{ focus: { user: { username: 'somebody' } } }]
+      );
 
       assert.isTrue(store.hasAppliedFilter());
```

Comment:
Can you clarify the user-facing effect of the changes here?

Reply:
Congratulations on finding a small bug for me! This is an artifact of mid-implementation and should be reverted. I shall do so.